plugins/in_windows_exporter_metrics/we_wmi.c

```diff
@@ -186,23 +184,21 @@ static double wmi_get_property_value(struct flb_we *ctx, char *raw_property_key,
     double val = 1.0;
     HRESULT hr;
     wchar_t *wproperty;
+    VARIANT dstProp;
 
     VariantInit(&prop);
+    VariantInit(&dstProp);
     wproperty = we_convert_str(raw_property_key);
     hr = class_obj->lpVtbl->Get(class_obj, wproperty, 0, &prop, 0, 0);
-    switch(prop.vt) {
-    case VT_I4:
-        val = prop.lVal;
-        break;
-    case VT_BSTR:
-        strprop = we_convert_wstr(prop.bstrVal, CP_UTF8);
-        wmi_utils_str_to_double(strprop, &val);
-        flb_free(strprop);
-        break;
-    default:
-        break;
+    VariantChangeType(&dstProp, &prop, 0, VT_BSTR);
```

Comment:
`VariantChangeType` allow in place conversion so `dstProp` is not necessary.

Reply:
Oh, I didn't know about that. I used in-place conversion for `VariantChangeType`.